### PR TITLE
fix: [M3-8142] - Alignment Regression Of The IP Address Inside Node Pools Table

### PR DESCRIPTION
**Pull request by carrillo-erik**
## Description 📝
Fixes the the vertical alignment of the IP Address inside of the Node Pools table. 

## Changes  🔄
List any change relevant to the reviewer.
- For testing purposes, I disabled the `background-color: transparent` css property on the `copy` icon in order to show the cause of the visual bug.
- ~~Updated the row width percentage values, allowing the `IP Address` column more room since some addresses will cause the text to wrap. The `Status` column does not need as much room and the values in these change maintain a similar proportion as before.~~
- Pass the `noWrap` prop set to `true` on the `<TableCell>` for the IP Address column to prevent longer addresses from wrapping the `copy` icon and resulting in vertical alignment discrepancies. 

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/119514965/b55cb5bf-018a-4a17-a537-599f8ca20d65" /> | <video src="https://github.com/linode/manager/assets/119514965/10ac1227-7686-43dc-8977-9c6d904eb2de" /> |

## How to test 🧪
- Open the web browser developer tools and modify the viewport width as shown in the `Preview` video to ensure consistent results. 

### Verification steps
(How to verify changes)
- Verify that as you adjust the viewport width, the IP Address which are wider than normal do not cause the copy icon to wrap to the next line (causing the IP Address to shift vertically).

## As an Author I have considered 🤔
*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ export const NodeTable = React.memo((props: Props) => {
                     <TableSortCell
                       sx={(theme) => ({
                         ...theme.applyTableHeaderStyles,
-                        width: '15%',
```
Comment (hkhalil-akamai) [inline packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx:103]: These changes are an improvement, but I'm still seeing the wrapping at very narrow viewport widths (i.e., mobile screens).

https://github.com/linode/manager/assets/122488130/87d28a05-f7e5-41e4-ba35-05c2cf67b75a

Comment (hkhalil-akamai) [reply packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx:103]: I was able to fix it by changing lines 114-117 of `NodeRow.tsx` like so:

```
          <div style={{ whiteSpace: 'nowrap' }}>
            <CopyTooltip copyableText text={displayIP} />
            <StyledCopyTooltip text={displayIP} />
          </div>
```

Using `whiteSpace: 'no-wrap'` ensures these elements will never wrap.

Comment (carrillo-erik) [reply packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx]: @hkhalil-akamai I believe the viewport width at which you reduced to in your video is somewhere in the 500px range and I was looking at the 600px breakpoint. In any event, I like the change you suggested better. We don't have to introduce a `<div>` element, we can just pass the `noWrap` prop set to `true` on the `<TableCell>` component.

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeRow.tsx [hunk 2]
```diff
@@ -100,7 +101,7 @@ export const NodeRow = React.memo((props: NodeRowProps) => {
           </>
         )}
       </TableCell>
-      <TableCell>
+      <TableCell noWrap={true}>
```
Comment (dwiley-akamai) [inline packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeRow.tsx:104]:
```suggestion
      <TableCell noWrap>
```

**Review by dwiley-akamai — APPROVED** [review]
IP Address and copy icon do not wrap as the viewport width shrinks ✅

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeRow.tsx (hunk 2 above)
Comment (hkhalil-akamai) [inline packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeRow.tsx:104]: Wow I didn't even know there was a `noWrap` prop! You can probably make it even more terse:

```suggestion
      <TableCell noWrap>
```

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx [hunk 3]
```diff
@@ -88,7 +88,7 @@ export const NodeTable = React.memo((props: Props) => {
                     <TableSortCell
                       sx={(theme) => ({
                         ...theme.applyTableHeaderStyles,
-                        width: '25%',
+                        width: '35%',
```
Comment (hkhalil-akamai) [inline packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx:91]: Even though the `noWrap` sorts out the alignment problem, I think the new widths you had before better fit the content (status column is wide)

**Review by hkhalil-akamai — APPROVED** [review]
Looks good, left some final feedback

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx (hunk 3 above)
Comment (carrillo-erik) [reply packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.tsx]: I've replaced the widths again.

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeRow.tsx (hunk 2 above)
Comment (carrillo-erik) [reply packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeRow.tsx]: Done!

Comment (carrillo-erik) [reply packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeRow.tsx]: 👍

**Pull request merged**